axsis.magix.py

```diff
@@ -1,68 +1,124 @@
+import abc
+import argparse
 import asyncio
 import json
 import os
 import time
 import sys
+import logging
 from functools import reduce
 
 import rx.operators as ops
 from magix_client import MagixHttpClient, Message
 from rx.scheduler.eventloop import AsyncIOScheduler
-
-from pi_device import create_pi_device
+from pi_device import create_pi_device, Mode
 
 kMagixHost = os.getenv('MAGIX_HOST', 'http://localhost:8080')
 kChannel = 'axsis-xes'
 
 
 class AxsisMessage:
-    def __init__(self, ip, action, value, port=50000):
+    def __init__(self, ip, action, value, port=50000, mode=Mode.simulation.name):
```

Comment:
Do I get it right that mode is now either `simulation` or passed via magix message?